.github/workflows/dev-fe-e2e.yaml

```diff
@@ -138,7 +138,7 @@ jobs:
       - name: Run integration tests
         shell: bash
         env:
-          EVEREST_BUCKETS_NAMESPACES_MAP: '[["bucket-1",["everest-ui"]],["bucket-2",["psmdb-only"]],["bucket-3",["pxc-only"]],["bucket-4",["pg-only"]]]'
+          EVEREST_BUCKETS_NAMESPACES_MAP: '[["bucket-1","my-special-place"],["bucket-2","psmdb-only"],["bucket-3","pxc-only"],["bucket-4","pg-only"]]'
```

Comment:
The e2e `EVEREST_CI_NAMESPACES` still have the `everest-ui` namespace in it: https://github.com/percona/everest/blob/2ad899853af7dbe963e71dc9d9bdee15b7c594ba/ui/apps/everest/.e2e/constants.ts#L7-L12  
Are you sure that `my-special-place` wasn't commited by a mistake?

Reply:
yes, it was, reverting it now. Thanks for noticing!